gui/src/elm/Address.elm

```diff
@@ -82,32 +81,39 @@ view helpers model =
             , ( "step-error", model.error /= "" )
             ]
         ]
-        [ p [ class "upper error-message" ]
-            [ text (helpers.t model.error) ]
-        , div [ class "upper" ]
-            [ input
-                [ placeholder (helpers.t "Address Cozy address")
+        [ div
+          [ class "step-content" ]
+          [ Icons.cozyBigIcon,
+          h1 [] [ text (helpers.t "Address Please introduce your cozy address") ]
+          , if model.error == "" then
+              p [ class"adress-helper"]
```

Comment:
Just saw the elm-format module for atom does not integrate with linting and have to be started manually. Will do now.